src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/Commands.DataFactoryV2.csproj

```diff
@@ -164,10 +164,13 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
     <Compile Include="Models\DataFactoryClient.Pipelines.cs" />
+    <None Include="AzureRM.DataFactoryV2.Netcore.psd1" />
```

Comment:
Remove this.

Reply:
This is our project file and the action is None for it. It just makes it easier for us to edit everything, why do you want this removed? It is not affecting anything.